lib/studies/reply/adaptive_nav.dart

```diff
@@ -876,17 +927,22 @@ class _BottomDrawerFolderSection extends StatelessWidget {
   }
 }
 
-class _MailNavigator extends StatelessWidget {
+class _MailNavigator extends StatefulWidget {
```

Comment:
I dont think this needs to be stateful

Reply:
I found that it does in this case, without it the `InboxPage()` would not rebuild when the destination changed.

Reply:
huh, that's very odd since there is no state being tracked

Reply:
Yeah, If I make it `Stateless`, and it give it a `key` then the `InboxPage` rebuilds but without any animation